imgcreate/dnfinst.py

```diff
@@ -92,14 +98,17 @@ def _cleanupRpmdbLocks(self, installroot):
         for f in glob.glob(installroot + "/var/lib/rpm/__db*"):
             os.unlink(f)
 
-    def setup(self, confpath, installroot, cacheonly=False, excludeWeakdeps=False):
+    def setup(self, confpath, installroot, cacheonly=False,
+              excludeWeakdeps=False, gpgcheck=False):
         self._writeConf(confpath, installroot)
         self._cleanupRpmdbLocks(installroot)
         self.conf.read(confpath)
         self.conf.installroot = installroot
         self.conf.prepend_installroot("cachedir")
         self.conf.prepend_installroot("persistdir")
         self.conf.install_weak_deps = not excludeWeakdeps
+        self.conf.gpgcheck = gpgcheck
+        self.conf.localpkg_gpgcheck = gpgcheck
```

Comment:
This option should not be set. Mostly because it won't do much, but I'd be concerned if it would accidentally break something.

Reply:
Yeah, I was kind of hoping it would enforce validation globally. It does not appear to.

Fixed in patch v2, 50b8f0ae90.